File: jeanstationfrontend/src/app/models/Notification.ts
export default class Notification {
    NotificationId: string;
    AddressId: string;
    UserId: string;
    OrderId: string;
    EmailSentDate: Date;
    constructor() {
        this.NotificationId = "";
        this.AddressId = "";
        this.UserId = "";
        this.OrderId = "";
        this.EmailSentDate = new Date();
    }
} 

File: NotificationsAPI/Models/Notification.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace NotificationsAPI.Models
{
    public class Notification
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public string NotificationID { get; set; }
        public string AddressId { get; set; }
        public string UserId { get; set; }
        public string OrderId { get; set; }
        public DateTime EmailSentDate { get; set; }    
    }
}
